certified-connectors/DocuSignDemo/script.csx

```diff
@@ -712,6 +712,14 @@ public class Script : ScriptBase
     var uriLogicAppsBase64 = Convert.ToBase64String(Encoding.UTF8.GetBytes(uriLogicApps ?? string.Empty));
     var notificationProxyUri = this.Context.CreateNotificationUri($"/webhook_response?logicAppsUri={uriLogicAppsBase64}");
 
+    var envelopeEventMap = new Dictionary<string, string>() {
+        {"Sent", "envelope-sent"},
```

Comment:
all these need to be made available in the drop down as well.
user should be able to use the new ones such as envelope-sent, envelope-delivered, ..envelope-voided etc